apps/meteor/tests/end-to-end/api/livechat/20-messages.ts

```diff
@@ -51,5 +52,17 @@ describe('LIVECHAT - messages', () => {
 				expect(quotedMessageAttachments).to.have.property('text').that.is.equal(agentMsgSentence);
 			}
 		});
+
+		it('it should verify if visitor is receiving a message with a image attachment', async () => {
+			const {
+				room: { _id: roomId },
+				visitor: { token },
+			} = await startANewLivechatRoomAndTakeIt();
+
+			const imgMessage = await uploadFile(roomId, token);
```

Comment:
I think it would be better if we could grab the message using this endpoint: https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/app/api/server/v1/chat.ts#L110

Reply:
ok, but the endpoint called by livechat is calling uploadFile...